lib/revision_feedback_service.rb

```diff
@@ -18,7 +18,7 @@ def feedback
   MINIMUM_REFERENCES = 1
   def citation_feedback
     ref_tags = @features['feature.wikitext.revision.ref_tags']
-    cite_templates = @features['feature.enwiki.revision.cite_templates']
+    # cite_templates = @features['feature.enwiki.revision.cite_templates']
```

Comment:
@ragesoss I have commented this out because I am not sure if the logic to determine citation_feedback needs to do check something against `cite_templates`. If that's not the case, we can remove this line.